<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>1176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dispertech: Config Tracking</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Locate</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Diameter</string>
        </property>
        <property name="buddy">
         <cstring>locate_diameter</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="locate_diameter"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Invert</string>
        </property>
        <property name="buddy">
         <cstring>locate_invert</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="locate_invert">
        <item>
         <property name="text">
          <string>True</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>False</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Min Mass</string>
        </property>
        <property name="buddy">
         <cstring>locate_minmass</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="locate_minmass"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Link</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Memory</string>
        </property>
        <property name="buddy">
         <cstring>link_memory</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="link_memory"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Search Range</string>
        </property>
        <property name="buddy">
         <cstring>link_search_range</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="link_search_range"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Filter</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Min Length</string>
        </property>
        <property name="buddy">
         <cstring>filter_min_length</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="filter_min_length"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Process</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Compute Drift</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>um / pixel</string>
        </property>
        <property name="buddy">
         <cstring>process_um_pixel</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Min Traj Len</string>
        </property>
        <property name="buddy">
         <cstring>process_min_traj_length</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Min mass</string>
        </property>
        <property name="buddy">
         <cstring>process_min_mass</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Max Size</string>
        </property>
        <property name="buddy">
         <cstring>process_max_size</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Max ecc</string>
        </property>
        <property name="buddy">
         <cstring>process_max_ecc</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>FPS</string>
        </property>
        <property name="buddy">
         <cstring>process_fps</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="process_um_pixel"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="process_min_traj_length"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="process_min_mass"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="process_max_size"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="process_max_ecc"/>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="process_fps"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="process_compute_drift">
        <item>
         <property name="text">
          <string>True</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>False</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_apply">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_reset">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
